mlir/lib/Dialect/Linalg/Transforms/TilingInterfaceImpl.cpp

```diff
@@ -324,7 +324,20 @@ struct LinalgOpTilingInterface
 // External Model for implementing `PartialReductionInterface` for `LinalgOp`s.
 //===----------------------------------------------------------------------===//
 
-/// External model implementation of PartialReductionInterface for LinalgOps.
+static AffineMap getPartialResultAffineMap(LinalgOp linalgOp,
```

Comment:
I think this needs more comments. This is assuming that you are doing outer-reduction/inner-parallel. So then it make sense to add as many dimensions here as the number of reduction dimensions to the end of the map. I think this needs to be called out. Comments here would be really useful. This part of the code is really involved and has been a pain to understand since it isnt very well commented.

Reply:
Added some comments explaining how it works.